test/helpers/stdio.js

```diff
@@ -1,10 +1,17 @@
-export const getStdinOption = stdioOption => ({stdin: stdioOption});
-export const getStdoutOption = stdioOption => ({stdout: stdioOption});
-export const getStderrOption = stdioOption => ({stderr: stdioOption});
-export const getStdioOption = stdioOption => ({stdio: ['pipe', 'pipe', 'pipe', stdioOption]});
-export const getInputOption = input => ({input});
-export const getInputFileOption = inputFile => ({inputFile});
-
-export const getScriptSync = $ => $.sync;
+import process from 'node:process';
 
 export const identity = value => value;
+
+export const getStdio = (indexOrName, stdioOption) => {
+	if (typeof indexOrName === 'string') {
+		return {[indexOrName]: stdioOption};
+	}
+
+	const stdio = ['pipe', 'pipe', 'pipe'];
+	stdio[indexOrName] = stdioOption;
+	return {stdio};
+};
```

Comment:
Replaced `getStdinOption()`, and so on. This is more generic and results in simpler tests.